.changeset/hungry-terms-hunt.md

```diff
@@ -1,6 +0,0 @@
----
-'@shopify/ui-extensions': minor
```

Comment:
@mleandres was this meant to be a minor bump? We actually can't do minor bumps for stable release (see [doc](https://github.com/Shopify/ui-extensions/blob/unstable/documentation/versions-and-deploys.md#adding-code)). Do you mind revising this changeset before we do a new release.

Reply:
@mleandres, right. Ok, so you should create another PR that changes that changset file, merge that into 2023-07 then  I think this PR should right itself after that commit.